extensions/awt/deployment/src/main/java/io/quarkus/awt/deployment/AwtProcessor.java

```diff
@@ -31,181 +36,242 @@ void nativeImageFeatures(BuildProducer<NativeImageFeatureBuildItem> nativeImageF
         nativeImageFeatures.produce(new NativeImageFeatureBuildItem(DarwinAwtFeature.class));
     }
 
-    @BuildStep(onlyIf = NativeBuild.class)
+    @BuildStep(onlyIf = NativeOrNativeSourcesBuild.class)
     UnsupportedOSBuildItem osSupportCheck() {
         return new UnsupportedOSBuildItem(WINDOWS,
                 "Windows AWT integration is not ready in native-image and would result in " +
                         "java.lang.UnsatisfiedLinkError: no awt in java.library.path.");
     }
 
-    @BuildStep(onlyIf = NativeBuild.class)
+    @BuildStep(onlyIf = NativeOrNativeSourcesBuild.class)
     NativeMinimalJavaVersionBuildItem nativeMinimalJavaVersionBuildItem() {
         return new NativeMinimalJavaVersionBuildItem(11, 13,
                 "AWT: Some MLib related operations, such as filter in awt.image.ConvolveOp will not work. " +
                         "See https://bugs.openjdk.java.net/browse/JDK-8254024");
     }
 
     @BuildStep(onlyIf = NativeOrNativeSourcesBuild.class)
-    void i18NProperties(
+    void resources(
             BuildProducer<NativeImageResourcePatternsBuildItem> resourcePatternsBuildItemBuildProducer) {
         resourcePatternsBuildItemBuildProducer
-                .produce(NativeImageResourcePatternsBuildItem.builder().includePattern(".*/iio-plugin.*properties$").build());
+                .produce(NativeImageResourcePatternsBuildItem.builder()
+                        .includePattern(".*/iio-plugin.*properties$") // Texts for e.g. exceptions strings
+                        .includePattern(".*/.*pf$") // Default colour profiles
```

Comment:
Please double check with `-H:Log=registerResource:3` whether or not this is in fact needed. It seemed to get included by default for my builds when testing https://github.com/Karm/mandrel-integration-tests/pull/146

Reply:
@jerboaa Mandrel Integration Tests don't stress this path at all. The AWT TS here contains a very robust test bank, e.g. https://github.com/quarkusio/quarkus/blob/main/integration-tests/awt/src/test/resources/encoders_test_config.txt

It is in fact needed.

The way I work on PRs like this is that I change things one by one until it passes the test suite, i.e. it fails several encoders tests without it. I stepped it in gdb and I also did some printf "debugging" to confirm and the encoders tests fail due to missing colour profiles files.

That debug session also lead me to an idea of providing my own colour profile files and it will come in a separate PR as I keep having troubles to make it work with Quarkus even in HotSpot mode (works in a simple non Q app though).

Reply:
OK. Thanks for checking!